solutions/devsprint-bruno-ramos-2/DeliveryAppChallengeTests/Modules/Product/RestaurantList/RestaurantListRequestTests.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  RestaurantListRequestTests.swift
+//  DeliveryAppChallengeTests
+//
+//  Created by Gustavo Tiecker on 31/01/22.
+//
+
+import Foundation
+import XCTest
+
+@testable import DeliveryAppChallenge
+
+final class RestaurantListRequestTests: XCTestCase {
+    
+    private let pathURLStub = "home_restaurant_list.json"
+    private let httpMethodStub: HTTPMethod = .get
+    
+    private lazy var sut = RestaurantListRequest()
```

Comment:
Como nosso objeto sobre testes não utiliza nenhum escopo da classe, ele pode ser um `let` ao invés de um `lazy var`
```suggestion
    private let sut = RestaurantListRequest()
```